package.json

```diff
@@ -228,5 +228,9 @@
     "@pulumi/pulumi": "3.19.0",
     "@pulumi/aws": "4.28.0"
   },
-  "packageManager": "yarn@3.1.0"
+  "packageManager": "yarn@3.1.0",
```

Comment:
1. These new dependencies should not be defined here, but in the actual package / `package.json` that is using them. 2. `https` is a native Node module, doesn't need to be included in list of dependencies.

Reply:
so I added this due to an error thrown by adio: 
```
✖ "adio" found some errors. Please fix them and try committing again.
1. @webiny/project-utils (/Users/ewan/Development/webiny/webiny-js/packages/project-utils)
Packages used in source code, but not listed in package.json:
1. https
```